package.json

```diff
@@ -33,10 +33,10 @@
     "test:coverage-90": "tap test/{unit,acceptance}/{*,**/*}.test.js --coverage --branches=90 --functions=90 --lines=90 --statements=90 --nyc-arg=\"--exclude=api\"",
     "test:coverage-report": "tap test/{unit,acceptance}/{*,**/*}.test.js --coverage --branches=90 --functions=90 --lines=90 --statements=90 --nyc-arg=\"--exclude=api\" && nyc report --reporter=text-lcov > coverage.lcov",
     "test:coverage-ui": "tap test/{unit,acceptance}/{*,**/*}.test.js --coverage --coverage-report=html --nyc-arg=\"--exclude=api\"",
-    "lint": "standard",
-    "lint:fix": "standard --fix",
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
     "license-checker": "license-checker --production --onlyAllow='MIT;Apache-2.0;Apache1.1;ISC;BSD-3-Clause;BSD-2-Clause'",
-    "build-esm": "npx gen-esm-wrapper . index.mjs && standard --fix index.mjs"
+    "build-esm": "npx gen-esm-wrapper . index.mjs && eslint --fix index.mjs"
```

Comment:
Are we missing the `.` here: `eslint . --fix`?

Reply:
I think it only fix one file index.mjs so we don't need '.'
`eslint .` is to fix everything besides what are in `.eslintignore`, right?

Reply:
Yes you are correct, I completely glossed over that.